### PR TITLE
pull request for early allocation instructions

### PR DESCRIPTION
**Pull request by mcr**
text as discussed on the list.

### CODE REVIEW

File: draft-ietf-core-sid-latest.md [hunk 1]
```diff
@@ -517,6 +517,54 @@ The allocation policy is Expert review. The Expert MUST ensure that the followin
...
+
+Due to the difficulty in changing SID values during IETF document processing, it is expected that most documents will ask for SID allocations using Early Allocations {{-BCP100}}.   
+The details of the Early Allocation should be included in any Working Group Adoption call.
+
+During the early use of SIDs, many existing, previously published YANG modules will not have SID allocations.  
+For an allocation to be useful the included YANG modules may also need to have SID allocations made.
+
+The Expert Reviewer who performs the (Early) Allocation analysis will need to go through the list of included YANG modules and perform SID allocations for those modules as well.
+
+If the document is a published RFC, then the allocation is permanent.
```
Comment (ivajloip) [inline draft-ietf-core-sid-latest.md]: I am not sure it is clear enough what is permanent - the SID allocation I suppose, so this is what is added in my version. Also formatted as bullet points in order to connect tightly coupled paragraphs together IMO improving readability

File: draft-ietf-core-sid-latest.md [hunk 2]
```diff
@@ -517,6 +517,54 @@ The allocation policy is Expert review. The Expert MUST ensure that the followin
...
+The Expert Reviewer provides the generated SID file to IANA for registration.
+This process may be someone busy during a bootstrap period (there are over 100 YANG modules to date, none of which have SID allocations), but should quiet down once needed entries are allocated.
+
+If the document is an unprocessed Internet-Draft adopted in a WG, then an Early Allocation is performed for this document as well.
+Early Allocations require approval by an IESG Area Director.
+An early allocation which requires additional allocations will list the other allocations in it's description, and will be cross-posted to the any other working group mailing lists.
```
Comment (ivajloip) [inline draft-ietf-core-sid-latest.md]: it's -> its

**Comment by ivajloip** [comment]
> I don't really understand the use of branches when you are the only active
author.

I guess it's a habit really - work on something until I am happy with it in a branch and then merge it (possibly rewriting history and all that if needed). 

I will see what Carsten had suggested for the 63bit SIDs and update my branch with it. If you are happy with the content of the branch [sid-09][1], I can merge it and publish a new version of the sid draft tomorrow.

[1]: https://github.com/core-wg/yang-cbor/tree/sid-09

**Comment by mcr** [comment]
Ivaylo Petrov <notifications@github.com> wrote:
    > I will see what Carsten had suggested for the 63bit SIDs and update my
    > branch with it. If you are happy with the content of the branch
    > [sid-09][1], I can merge it and publish a new version of the sid draft
    > tomorrow.

Please go ahead.

**Comment by ivajloip** [comment]
I published a new version of the sid draft and merged everything to master. If some of the points I modified still needs clarification, don't hesitate to let me know or create a new PR.

**Pull request closed without merging**